Vizzuality/climate-watch
# Feature/import ndc content

- adds Location model to represent countries & country groups
- adds Ndc model to represent NDC document content
- integrates AWS S3 storage used for NDC .md documents and media files
- adds a service object and rake task to import documents from S3 into the ndcs table
- at this point it does not yet index documents for full text search, thought I'd separate that

To test this:
- you need the `.env` set up correctly to access the S3 bucket
- `rake db:migrate`
- create some location manually, as location seeding is not yet in place: `Location.create(code: 'AFG', wri_standard_name: 'Afghanistan')` (needs to match a code in the S3 folder, where md files generally have the form `ndcs/[code](-optional).md`)
- `bundle exec rake ndc_content:import` - should create an Ndc for Afghanistan with md content in the `content` column
